requirements.txt

```diff
@@ -0,0 +1,129 @@
+alembic==0.9.3.dev0
+apturl==0.5.2
+asn1crypto==0.24.0
+astroid==1.6.4
+atomicwrites==1.1.5
+attrs==18.1.0
+Babel==2.4.0
+beautifulsoup4==4.6.0
+blinker==1.4
+Brlapi==0.6.6
+certifi==2018.1.18
+chardet==3.0.4
+checkbox-support==0.22
+click==6.7
+colorama==0.3.7
+command-not-found==0.3
+cryptography==2.1.4
+cupshelpers==1.0
+defer==1.0.6
+distro-info==0.18
+feedparser==5.2.1
+Flask==0.12.2
+Flask-BabelEx==0.9.3
+Flask-Gravatar==0.4.2
+Flask-HTMLmin==1.3.2
+Flask-Login==0.4.0
+Flask-Mail==0.9.1
+Flask-Migrate==2.1.1
+Flask-Paranoid==0.2.0
+Flask-Principal==0.4.0
+Flask-Security==1.7.5
+Flask-SQLAlchemy==2.1
+Flask-WTF==0.14.2
+guacamole==0.9.2
+html5lib==0.999999999
+htmlmin==0.1.12
+httplib2==0.9.2
+idna==2.6
+isort==4.3.4
+itsdangerous==0.24
+Jinja2==2.10
+keyring==10.6.0
+keyrings.alt==3.0
+language-selector==0.1
+launchpadlib==1.10.6
+lazr.restfulclient==0.13.5
+lazr.uri==1.0.3
+lazy-object-proxy==1.3.1
+louis==3.5.0
+lxml==4.2.1
+macaroonbakery==1.1.3
+Mako==1.0.7
+MarkupSafe==1.0
+mccabe==0.6.1
+more-itertools==4.2.0
+oauth==1.0.1
+oauthlib==2.0.6
+olefile==0.45.1
+onboard==1.4.1
+padme==1.1.1
+passlib==1.7.1
+pexpect==4.2.1
+Pillow==5.1.0
+pipenv==2018.5.18
+plainbox==0.25
+pluggy==0.6.0
+protobuf==3.0.0
+psycopg2==2.7.4
+py==1.5.3
+pyasn1==0.4.2
+pycairo==1.16.2
+pycrypto==2.6.1
+pycups==1.9.73
+pycurl==7.43.0.1
+pygobject==3.26.1
+pyinotify==0.9.6
+PyJWT==1.5.3
+pylint==1.9.1
+pymacaroons==0.13.0
+PyNaCl==1.1.2
+pyOpenSSL==17.5.0
+pyparsing==2.2.0
+pyRFC3339==1.0
+pytest==3.6.0
+python-apt==1.6.0
+python-dateutil==2.6.1
+python-debian==0.1.32
+pytz==2018.3
+pyxdg==0.25
+PyYAML==3.12
+reportlab==3.4.0
+requests==2.18.4
+requests-unixsocket==0.1.5
+SecretStorage==2.3.1
+simplejson==3.13.2
+six==1.11.0
+SQLAlchemy==1.1.11
+sqlparse==0.2.4
+system-service==0.3
+systemd-python==234
+ubuntu-drivers-common==0.0.0
+ufw==0.35
+unattended-upgrades==0.1
+unity-scope-calculator==0.1
+unity-scope-chromiumbookmarks==0.1
+unity-scope-colourlovers==0.1
+unity-scope-devhelp==0.1
+unity-scope-firefoxbookmarks==0.1
+unity-scope-manpages==0.1
+unity-scope-openclipart==0.1
+unity-scope-texdoc==0.1
+unity-scope-tomboy==0.1
+unity-scope-virtualbox==0.1
+unity-scope-yelp==0.1
+unity-scope-zotero==0.1
+unity-tweak-tool==0.0.7
+urllib3==1.22
+usb-creator==0.3.3
+virtualenv==16.0.0
+virtualenv-clone==0.3.0
+wadllib==1.3.2
+webencodings==0.5
+Werkzeug==0.14.1
+wrapt==1.10.11
+WTForms==2.1
+xdiagnose==3.8.8
+xkit==0.0.0
+XlsxWriter==0.9.6
+zope.interface==4.3.2
```

Comment:
Make sure you only have modules needed to run this project in your `requirement.txt` file to avoid time wastage when people try to install your project.

Reply:
I have rectified the issue